### DISCUSSION TITLE
whats the correct way to stop the bot without stoping the program

### DISCUSSION THREAD

**Discussion by iDhanush** [aiogram/aiogram#1229] [General]
## Context

i want to stop a bot without stoping the program
the bot is webhooked using fastapi

i tried all the below functions and it didn;t do the job

`await bot.delete_webhook()

await (await bot.get_session()).close()

await dp.storage.close()

await dp.storage.close()`

**Comment by Olegt0rr** [comment]
`.delete_webhook()` - will ask Telegram Bot API server to stop sending webhooks to your endpoint

Then you may stop related connections, workers, storages, e.t.c.
If you have some issues with it – please, provide reproducible example
